python/triton/compiler/code_generator.py

```diff
@@ -693,27 +722,26 @@ def visit_if_top_level(self, cond, node):
             # then terminator
             self.builder.set_insertion_point_to_end(then_block)
             assert not then_block.has_terminator(), f"{then_block}"
-            then_handles = [then_defs[n]._flatten_ir() for n in names]
-            then_handles_spec, then_handles_flat = list_list_flatten(then_handles)
-            self.builder.create_branch(endif_block, then_handles_flat)
+            then_handles = flatten_values_to_ir(then_defs[name] for name in names)
+            self.builder.create_branch(endif_block, then_handles)
```

Comment:
These control-flow changes are purely cleanup to match the new calling convention for `_flatten_ir` and `_unflatten_ir`